src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
```

Comment:
![image](https://github.com/mate-academy/react_autocomplete/assets/56752562/c66f5c1c-2e4d-4a55-b335-08218dafa7b9)
Fix or remove it.

Reply:
'lodash.debounce' should be listed in the project's dependencies. Run 'npm i -S lodash.debounce' to add it